src/System.Net.Http/tests/FunctionalTests/HttpClientHandlerTest.Http2.cs

```diff
@@ -1111,7 +1111,8 @@ public async Task GoAwayFrame_AbortAllPendingStreams_StreamFailWithExpectedExcep
                     await newConneciton.SendDefaultResponseHeadersAsync(retriedStreamId).ConfigureAwait(false);
                     await newConneciton.SendResponseDataAsync(retriedStreamId, new byte[0], endStream: true).ConfigureAwait(false);
 
-                    await newConneciton.WriteFrameAsync(goAwayFrame).ConfigureAwait(false);
+                    var goAwayFrameOnNewConnection = new GoAwayFrame(retriedStreamId, (int)ProtocolErrors.ENHANCE_YOUR_CALM, new byte[0], 0);
```

Comment:
btw. I have previously sent GOAWAY with 0 but that happened after we have already sent complete response - currently client assumes that server has processed the stream completely but server claimed with GOAWAY that it hasn't - it feels like this is something spec should clarify further as I'm not sure if the client should somehow retroactively thrown in that case.